src/plugins/kibana_react/public/page_template/no_data_page/no_data_card/no_data_card.tsx

```diff
@@ -25,10 +31,14 @@ export const NoDataCard: FunctionComponent<NoDataPageActions> = ({
   return (
     <EuiCard
       paddingSize="l"
-      title={title}
+      title={title || defaultCTATitle}
+      description={i18n.translate('kibana-react.noDataPage.noDataCard.description', {
+        defaultMessage: `Proceed without collecting data`,
+      })}
       betaBadgeLabel={recommended ? NO_DATA_RECOMMENDED : undefined}
       footer={footer}
-      {...(cardRest as any)}
+      layout={layout as 'vertical' | undefined}
+      {...cardRest}
```

Comment:
@chandlerprall Looks like our `any` is not longer a valid type anymore 😂 ...

I've been going back and forth with @jloleysens about this particular case where I'm just trying to extend EuiCard, passing on the required props to the consumers of this Kibana specific NoDataCard. I'd rather not have any default `title` or `description` here mainly because it really **needs** to be provided by the consumer; no default will work for all cases.

Do you have any tricks you can think of that might help us here?

Reply:
We have a couple of options here:

1. Update `NoDataPageActions` to not use `Partial<EuiCardProps>` (or just make `title` required as it is in `EuiCardProps`)
2. Don't change external contract and just ignore `undefined` values by doing `title!`

Either way we should be clear on what the intention is for consumers. It sounds like what we probably should do is option 1.

Reply:
Ugh, I really wish TS would fix how they handle spreading rests. Sounds like option 1 is the best path forward, but I don't know how many code locations would need to be updated to accommodate that?

Reply:
OK, so it looks like there are a couple of places we'll have to update. Mostly test files, thankfully, and two points where the component is being used in the wild.

I think, to scope this PR down, I'll implement option 2 and then create an issue after merging to address the point that @cchaos raised. Does that sound good?

Reply:
Yeah I'd actually prefer option 2. Thanks @jloleysens !!